app/components/works/show/attached_file_component.html.erb

```diff
@@ -1,10 +1,10 @@
 <tr>
-    <td><%= filename %>
+    <td class="file-level" style="--depth: <%=depth+1%>rem"><li><%= basename %>
```

Comment:
For consistency (until we merge the erb-lint PR), can you add spaces between `depth+1` and `<%%>` ?